pkg/insight/dto/chunk.go

```diff
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package insight
+package dto
```

Comment:
@sanposhiho I understand why you named this as "dto". And as I see, this PR
- moves "store" code to "insight/insightstore" package
- moves "model" code to "insight/dto" package
- keeps utility functions in "insight" package

, right?

But how do you think about this
- moves "store" code to "insight/insightstore" package
- keeps "model" code and utility functions in "insight" package

I think this is simpler.

Reply:
Agree 👌 Moved files from dto to insight.